api/lang/en/c_suite_role_title.php

```diff
@@ -3,15 +3,14 @@
 use Illuminate\Support\Facades\Lang;
 
 return [
-    'chief_audit_officer' => 'Chief Audit Officer',
-    'chief_finance_officer' => 'Chief Finance Officer',
+    'chief_audit_officer' => 'Chief Audit Executive',
+    'chief_finance_officer' => 'Chief Financial Officer',
     'chief_data_officer' => 'Chief Data Officer',
     'chief_digital_officer' => 'Chief Digital Officer',
     'chief_information_officer' => 'Chief Information Officer',
     'chief_privacy_officer' => 'Chief Privacy Officer',
     'chief_technology_officer' => 'Chief Technology Officer',
-    'deputy_chief_audit_officer' => 'Deputy Chief Audit Officer',
-    'deputy_chief_finance_officer' => 'Deputy Chief Finance Officer',
+    'deputy_chief_finance_officer' => 'Deputy Chief Financial Officer',
```

Comment:
Shouldn't the key be `deputy_chief_financial_officer` to match the value?

Reply:
Fair enough. Done in [1a45bb6](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13027/commits/1a45bb670356fd337ddfa3547bdb3c3ac5047618).